doc/source/whatsnew/v0.18.2.txt

```diff
@@ -89,6 +89,9 @@ Other enhancements
 
 - ``pd.read_html()`` has gained support for the ``decimal`` option (:issue:`12907`)
 
+- ``eval``'s upcasting rules for float32 types have also been updated to be more
```

Comment:
double-backticks around ``float32`` (and ``float64``)

no space after the colon ``(:issue:`12388`)``